### PR TITLE
[UI] Add tabs wrapping on Product show page > Attributes section

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         |  1.7
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

before:
image

fixed: 
image

### CODE REVIEW

**Review by Tomanhez — APPROVED** [review]

File: src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss [hunk 1]
```diff
@@ -202,6 +202,10 @@ a {
     border: 1px solid rgba(34,36,38,.1);
 }
 
+.ui.top.attached.tabular.menu{
```
Comment (GSadee) [inline src/Sylius/Bundle/AdminBundle/Resources/private/sass/_ui.scss:205]:
```suggestion
.ui.top.attached.tabular.menu {
```

**Review by GSadee — CHANGES_REQUESTED** [review]
As it is a bug fix, I would be for changing the branch to 1.7.

And could you add some screenshots to show how the product show page looks after the change?

**Comment by kulczy** [comment]
it would be a bit clearer to put it under the `tabs` comment rather than `menu`

**Comment by lchrusciel** [comment]
We need to resolve https://github.com/Sylius/Sylius/pull/11427 to merge this PR

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Probably rebase should fix the build

**Review by pamil — APPROVED** [review]
Build will be fixed by #11430.

**Comment by pamil** [comment]
Thank you, @SirDomin! :tada:

**Pull request merged**